lib/utilities/paginator.rb

```diff
@@ -0,0 +1,106 @@
+module Utilities
+  module Paginator
+    def items_to_display(model:, items_per_page: 5, current_page: 1)
+      @items_per_page = items_per_page
+      @current_page = current_page.to_i
```

Comment:
Ah yeah, it's because in my BetsController I pass the parameter current_page: params[:page] as the argument. So, in this case it's either I apply it here just to be safe and make sure it's always an int or I can pass it there.
Please let me know what works best.